UDPBroadcast/UDPBroadcastConnection.swift

```diff
@@ -80,6 +81,20 @@ open class UDPBroadcastConnection {
             return false
         }
         
+        if bindIt {
+            var saddr = sockaddr(sa_len: 0, sa_family: 0,
+                                 sa_data: (0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0))
+            self.address.sin_addr = INADDR_ANY
+            memcpy(&saddr, &self.address, MemoryLayout<sockaddr_in>.size)
+            self.address.sin_addr = INADDR_BROADCAST
+            let binded = bind(newSocket, &saddr, socklen_t(MemoryLayout<sockaddr_in>.size))
```

Comment:
Sorry to be nitpicky here but the correct naming of the `let` constant should be `isBound`.

Reply:
👍🏻

Reply:
updated in last commit